src/api/collections/CollectionList.ts

```diff
@@ -0,0 +1,74 @@
+import { ApolloQueryResult } from "apollo-client";
+import {
+  CollectionList as CollectionListQuery,
+  CollectionListVariables,
+} from "../../queries/gqlTypes/CollectionList";
+import { BaseCollection } from "../../fragments/gqlTypes/BaseCollection";
+import { PageInfo } from "../../fragments/gqlTypes/PageInfo";
+import { List } from "../types";
+
+export const COLLECTIONS_PER_API_CALL = 20;
```

Comment:
Such variables would be passed in SDK config, optionally obtained from default config.